clients/drcachesim/tests/burst_replaceall.cpp

```diff
@@ -42,11 +42,24 @@
 #include "drmemtrace/drmemtrace.h"
 #include "drvector.h"
 #include "../../../suite/tests/client_tools.h"
+#include "../../../suite/tests/condvar.h"
 #include <assert.h>
 #include <iostream>
 #include <math.h>
 #include <stdlib.h>
 #include <string.h>
+#include <set>
+#ifndef WINDOWS // XXX i#2040: Static client limitations
```

Comment:
This is confusing. There's an ifndef followed by an ifdef for the same symbol. Is that because of #2040? I don't understand how this works though.

Reply:
The idea was to be able to re-enable for Windows by just removing these marked ifndefs which is why the lines inside may contradict the outside.  What is your suggestion: remove these lines entirely (though if re-enabled someone will have to figure out that this include is missing from the build error)?

Reply:
A comment should be enough.

Reply:
Done.